docs/tre-admins/auth.md

```diff
@@ -1,30 +1,32 @@
 # Authentication and authorization
 
-[Azure Active Directory (AAD)](https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/active-directory-whatis) is the backbone of Authentication and Authorization in the Trusted Research Environment.
-
-AAD holds the identities of all the TRE/workspace users, including administrators, and connects the identities with applications which define the permissions for each user role.
+[Azure Active Directory (AAD)](https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/active-directory-whatis) is the backbone of Authentication and Authorization in the Trusted Research Environment. AAD holds the identities of all the TRE/workspace users, including administrators, and connects the identities with applications which define the permissions for each user role.
 
 ## Pre-requisites
-The following values are needed to be in place to run the script. (`/templates/core/.env`)
+The following values are needed to be in place before you run the creation process. (`/templates/core/.env`)
 
 | Key | Description |
 | ----------- | ----------- |
-|TRE_ID|This is needed to build up the redirect URI for the Swagger App|
+|TRE_ID|This is used to build up the name of the identities|
 |AAD_TENANT_ID|The tenant id of where your AAD identities will be placed. This can be different to the tenant where your Azure resources are created.|
-|AUTO_WORKSPACE_APP_REGISTRATION| Default of `false`. Setting this to true grants the `Application.ReadWrite.All` permission to the API AAD Application. This allows it to create other AAD applications, e.g. Workspaces.
+| LOCATION | Where your Azure assets will be provisioned. This is actually used to add a redirect URI from the Swagger UI to the API Application.
```

Comment:
No need for `actually` in `This is actually...`

Is LOCATION an azure Location (such as "West Europe" or "westeurope")?